### ISSUE TITLE
Subsolid pulmonary nodule

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6635]
**Preferred term label:**Subsolid pulmonary nodule

**Synonyms**
SNN

**Definition (free text, please give PubMed ID)**
A pulmonary nodule with at least partial ground-glass appearance

**Parent term (use hpo.jax.org/app)**
Pulmonary nodule (once created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik 
amplifiying the definition for non-specialists as follows, please reopen if not OK

Pulmonary subsolid nodules (SSNs) refer to pulmonary nodules with pure ground-glass nodules and part-solid ground-glass nodules. A ground-glass nodule (GGN) is the morphologic description of a pulmonary nodule category on thin-section chest computed tomography (CT). During the past decade, the natural history, management strategy and long-term prognosis in the case of GGNs have attracted attention. Pure GGNs are defined as focal nodular areas of increased lung attenuation through which lung parenchymal structures, such as the pulmonary vessels or bronchial structures, can be observed. Part-solid GGNs are nodules that present with both ground-glass and solid components in which the underlying lung architecture cannot be visualized. The term subsolid nodules (SSNs) includes both pure and part-solid GGNs.
https://pubmed.ncbi.nlm.nih.gov/24100062/
